src/pages/Home/Home.css

```
@@ -1,101 +1,96 @@
+.some-section {
+    width: min(80%, 1000px);
+    margin: 0 auto;
+}
+
 .card-container {
     display: flex;
-    justify-content: center;
+    justify-content: space-between;
+    gap: 8px;
     flex-wrap: wrap;
-    margin: 4%;
-    max-width: 100%;
-    position: relative;
+    width: 100%;
 }
 
 /* mission statement section */
+
+.mission-statement-container {
+    width: min(80%, 1000px);
+    margin: 48px auto;
+}
+
 .mission-statement-title {
     text-align: center;
     font-weight: 700;
     font-size: 2em;
     letter-spacing: 0.025em;
-    padding-top: 125px;
+    position: relative;
+    margin: 32px 0;
+    padding: 16px 0;
 }
 
-.mission-statement-container {
-    display: flex;
-    justify-content: center;
-    flex-wrap: wrap;
-    max-width: 75%;
-    position: relative;
-    margin: auto;
-    overflow: hidden;
+.mission-statement-text p + p {
+    margin-top: 16px;
 }
 
 .title-underline {
-    max-width: 100%; 
-    width: 7em;
-    border-bottom: 1px solid #77B0F3; 
-    line-height: 0.1em;
-    margin: -15px 20em 1.25em; 
+    position: absolute;
+    width: 20%;
+    height: 3px;
+    bottom: 0;
+    left: 50%;
+    transform: translateX(-50%);
+    background-color: #7C9DAF;
```

Please change the color to match the Figma design:

```suggestion
    background-color: #77B0F3;
```

---

Ok, I must have copied the wrong thing